### ISSUE TITLE
[Main UI] Issues in Regional Settings page

### ISSUE THREAD

**Issue by MHerbst** [openhab/openhab-webui#343]
In my first tests of the new UI I have found some issues (tested in Chrome Browser on Windows 10 with german language):

Language selection contains two empty entries at the beginning:
![image](https://user-images.githubusercontent.com/616792/93811897-22865180-fc51-11ea-9e42-d161a512caca.png)

BTW: is it really necessary to have languages like "Altpreußisch", "Kölsch" etc. in the list?

Country/Region also contains two empty entries:
![image](https://user-images.githubusercontent.com/616792/93812028-5c575800-fc51-11ea-9d40-7b9b81b7054c.png)

Below the Country/Region is this text:
![image](https://user-images.githubusercontent.com/616792/93812091-709b5500-fc51-11ea-93e1-06a1eaeef486.png)
Seems to be copied from the old Paper UI, but in my opinion it does not make sense here.

"Variant" has also two empty entries at the beginning of the list.

Use "Encoding" instead of "Script":
The name "Script" seems unusual to me. I have never seen it in this context. In my opinion "Encoding" would be better.

**Comment by ghys** [comment]
This has been the case since forever and it's not a problem with the UI.
The UI adds an empty option because the parameter is `required: false` but the list of options also supply an explicit empty one:

![image](https://user-images.githubusercontent.com/2004147/93858637-c527ea00-fcbc-11ea-9bf9-b5a35d559ca6.png)

This explicit empty option should be removed.

Descriptions are also set in the config description, the UI merely displays them.

**Comment by MHerbst** [comment]
> This has been the case since forever and it's not a problem with the UI.

You are right, it is only more noticeable now because of the new representation.

> The UI adds an empty option because the parameter is required: false

I think it would be nice to have a text for the empty option, maybe with a special label text like "<empty>" to make clear that this selection can be used to set the value to empty.

@ghys  can you or @kaikreuzer move this issue to openhab-core?
